.docker/opt/configure-aiida.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+
+# This script is executed whenever the docker container is (re)started.
+
+#===============================================================================
+# debugging
+set -x
+
+#===============================================================================
+# environment
+reentry scan
+
+export SHELL=/bin/bash
+
+#===============================================================================
+# setup AiiDA
+aiida_backend=django
```

Comment:
Should this be configurable?

Reply:
Made all options configurable. It will also be useful for AiiDA lab when setting up a new user.